processor/src/main/java/org/mapstruct/ap/internal/processor/MapperCreationProcessor.java

```diff
@@ -320,8 +321,21 @@ else if ( method.isMapMapping() ) {
                 hasFactoryMethod = mapMappingMethod.getFactoryMethod() != null;
                 mappingMethods.add( mapMappingMethod );
             }
+            else if ( method.isValueMapping() ) {
+                // prefer value mappings over enum mapping
+                ValueMappingMethod valueMappingMethod = new ValueMappingMethod.Builder()
+                    .mappingContext( mappingContext )
+                    .souceMethod( method )
+                    .valueMappings( mappingOptions.getValueMappings() )
+                    .build();
+                mappingMethods.add( valueMappingMethod );
+            }
             else if ( method.isEnumMapping() ) {
 
+                messager.printMessage(
```

Comment:
Nice with the deprecation. Is there still a test in place to make sure it still works for the time being?